src/winget-pkgs-automation/packages/t/tencent.start.json

```diff
@@ -15,8 +15,7 @@
   "PreviousVersion": "0.11.0.9200",
   "ManifestFields": {
     "PackageVersion": "$Response.latestversion",
-    "InstallerUrls": "$Response.downloadurl",
-    "ReleaseDate": "$Response.updatedate.Replace('.', '-')"
```

Comment:
Why remove `ReleaseDate`? Does API returns incorrect date?

Reply:
Yeah it has been sticked to 2022-05-23 for several versions.